devel/sparsehash/Portfile

```diff
@@ -31,16 +30,24 @@ post-patch {
     reinplace "s|@VERSION@|${version}|g" ${worksrcpath}/configure.ac
 }
 
+# While the port installs only headers, it builds tests in the process.
+# Using noarch setting breaks the build: https://github.com/macports/macports-ports/pull/15768
+installs_libs       no
+
 use_autoreconf      yes
 autoreconf.cmd      ./autogen.sh
 autoreconf.args
-depends_build-append port:autoconf port:automake port:libtool
-
-# This is a header-only library.
-supported_archs     noarch
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:gperftools \
+                    port:libtool
 
 # Setting compiler.cxx_standard (to any value) is necessary so that MacPorts
 # sets configure.cxx_stdlib, which the port needs in order to build for the
 # right stdlib. I don't know what version of the C++ standard this port
 # requires, so I'll just guess.
 compiler.cxx_standard 1998
```

Comment:
`compiler.cxx_standard` only needed to be set when `supported_archs` was `noarch`; if that's is removed, this should be too.